### PR TITLE
S3 read urls

### PR DESCRIPTION
**Pull request by tosan88**
PAM will be configured to check different S3 location per region. 

The following etcd key should be configured: `/ft/config/monitoring/s3-urls`, having something like that: `env1:http://s3link1.org,env2:http://s3link1.org`

### CODE REVIEW

File: scheduler_test.go [hunk 1]
```diff
@@ -29,3 +33,87 @@ func TestValidType(testing *testing.T) {
...
+					"Image",
+				},
+			},
+		},
+		Threshold: 1,
+	}
+
+	var mockEnvironments = make(map[string]Environment)
+	s3ReadURL1 := "http://s1.example.org"
+	mockEnvironments["env1"] = Environment{"env1", "http://env1.example.org", s3ReadURL1, "user1", "pass1"}
```
Comment (tamas-molnar) [inline scheduler_test.go]: Can you extract the readURL in a var too pls? Same in the test below for s3ReadURL1

File: README.md [hunk 2]
```diff
@@ -84,6 +84,7 @@ __Note that deployment to FTP2 is no longer supported.__
 The monitor can check publication across several different environments, provided each environment can be accessed by a single host URL. The monitor reads from `etcd` and watches for changes in the following paths:
 
 `/ft/config/monitoring/read-urls`: a comma-separated list of _name_`:`_value_ pairs, mapping from environment name to base read URL, e.g. `env1:http://foo.example.org,env2:http://bar.example.org`
+`/ft/config/monitoring/s3-urls`: a comma-separated list of _name_`:`_value_ pairs, mapping from environment name to base S3 URL, e.g. `env1:http://foo.example.org,env2:http://bar.example.org`
```
Comment (tosan88) [inline README.md]: Done

File: scheduler_test.go (hunk 1 above)
Comment (tosan88) [inline scheduler_test.go]: Done

**Comment by tosan88** [comment]
@khatton-ft @tamas-molnar @kovacshuni FYI: Made some additional changes

**Comment by kovacshuni** [comment]
that was and is still a +1 from me.

**Pull request merged**